DOCS.md

```diff
@@ -11,440 +11,397 @@ Version 1.0.0
+
+#### Terminal Events
+
+The *terminal events* are:
+* `adCompleted`: the user has exited the true\[X\] ad
+* `userCancelStream`: the user intends to exit the current stream entirely
+* `noAdsAvailable`: there were no true\[X\] ads available to the user
+* `adError`: the renderer encountered an unrecoverable error
+
+It's important to note that the player should not immediately resume playback once receiving the `adFreePod` event -- rather, it should note that it was fired and continue to wait for a terminal event.
+
 
 ### Handling Ad Elimination
 
-Skipping video ads is completely the responsibility of the app code. The
-SSAI API should provide enough information for the app to determine
-where the current pod end-point is, and the channel, when appropriate,
-should fast-forward directly to this point when resuming playback.
+Skipping video ads is completely the responsibility of the channel code. The SSAI API should provide enough information for the channel to determine where the current pod end-point is, and the channel, when appropriate, should fast-forward directly to this point when resuming playback.
 
 ## TruexAdRenderer Roku API
 
-This is an outline of `TruexAdRenderer` input and output events. Input
-events are assigned on the `action` field from the interface of the
-`TruexAdRenderer`, while output events are emitted against the
-`event` field on the same component.
+This is an outline of `TruexAdRenderer` input and output events. Input events are assigned on the `action` field from the interface of the `TruexAdRenderer`, while output events are emitted against the `event` field on the same component.
 
-### Reference to Truex Component Library
+### Reference to the true\[X\] Component Library
 
-The TrueX interactive ad component and its rendering logic are
-distributed as part of a component library. It is required for the
-hosting channel to add reference to the component library in order for
-it to be used, for instance via the following reference added to your
-channel’s main scene:
+The true\[X\] interactive ad component and its rendering logic are distributed as part of a component library. It is required for the hosting channel to add a reference to the component library in order for it to be used, for instance via the following reference added to your channel’s main scene:
 
     <ComponentLibrary id="TruexAdLibrary" uri="http://static.truex.com.s3.amazonaws.com/roku/TruexAdRenderer-Roku-v0.9.0.pkg"/>
```

Comment:
Noticed this link is dead. static.truex.com s3 bucket doesn't have a `roku` folder. Is this a future spec we plan to use?

Reply:
I've added a JIRA issue to update this with the final resting place of the released package: https://truextech.atlassian.net/browse/ADLAB-1138